FusionIIIT/applications/scholarships/models.py

```diff
@@ -1,3 +1,150 @@
 # from django.db import models
 
 # Create your models here.
+from django.db import models
+from applications.academic_information.models import Student
+# Create your models here
+
+class Mcm(models.Model):
+    mcm_id              = models.IntegerField(primary_key=True)
+    student_id         = models.ForeignKeyField(Student, on_delete=models.CASCADE)
+    income_total        = models.IntegerField()
+    income_file         = models.FileField(max_length=10)
+    banck_acc           = models.CharField(max_length=15)
+    brother_name        = models.CharField(max_length=30)
+    brother_occupation  = models.TextField(max_length=100)
+    sister_name         = models.CharField(max_length=30)
+    sister_occupation   = models.TextField(max_length=100)
+    income_father       = models.IntegerField()
+    income_mother       = models.IntegerField()
+
+    FATHER_OCC_CHOICES  = (
+            ( 'Service',(
+                        ('government','Government'),
+                        ('private','Private'),
+                        ('public','Public'),
+                        ) 
+            ),
+            ('Non_Salaried',(
+                         ('business','Business'),
+                         ('medical','Medical'),
+                         ('consultant','Consultant'),
+                        )
+            ),
+            ('pensioners','Pensioners'),
+    )
+    father_occ_choice=models.CharField(max_length=10,choices=FATHER_OCC_CHOICES)
+    father_occ          = models.TextField(max_length=100)
+    EMPLOYED='EMP'
```

Comment:
no need to give variable names. You can directly use this in the dictionary of choices. Refer to models in global for reference.